src/Forms/FixCommonErrors.cs

```diff
@@ -4776,7 +4772,8 @@ private void ToolStripMenuItemDeleteClick(object sender, EventArgs e)
                         int number = Convert.ToInt32(item.SubItems[1].Text);
                         if (number > firstIndex)
                             number -= subtitleListView1.SelectedItems.Count;
-                        deSelectedFixes.Add(number + item.SubItems[2].Text + item.SubItems[3].Text);

```

Comment:
Note that `number` can be `>` than `firstIndex` but we are not sure if `subtitleListView1.SelectedItems.Count` if going to be `<` than number so  `number -= subtitleListView1.SelectedItems.Count` can be negative